net-im/psi/metadata.xml

```diff
@@ -1,16 +1,21 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
-	<longdescription>Psi is a very good jabber client that uses QT</longdescription>
+	<maintainer type="person">
```

Comment:
In addition you also need to add proxy-maint team, like so: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml